riscv/processor.cc

```diff
@@ -1115,6 +1123,13 @@ reg_t processor_t::get_csr(int which)
       if (!state.debug_mode)
         break;
       return state.dscratch1;
+    case CSR_CPUCTRL:
+      // cpuctrl[0] will read 0 unless Icache param in Ibex is set.
+      // cpuctrl[5:1] will read 0 unless SecureIbex param in Ibex is set.
+      return (secure_ibex ? state.cpuctrl & reg_t(0x3e) : reg_t(0)) |
+             (icache_en   ? state.cpuctrl & reg_t(0x1)  : reg_t(0));
+    case CSR_SECURESEED:
+      return (reg_t)0;
```

Comment:
Nit: Maybe move these to the end of the switch? It doesn't look like the existing entries are in order anyway, so maybe it makes more sense to have "Ibex-specific stuff" grouped at the bottom?

Reply:
Sure, can definitely move them to the end.